demos/http/http_demo_basic_tls/http_config.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef HTTP_CONFIG_H
+#define HTTP_CONFIG_H
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Logging related header files are required to be included in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define LIBRARY_LOG_NAME and  LIBRARY_LOG_LEVEL.
+ * 3. Include the header file "logging_stack.h".
+ */
+
+/* Include header that defines log levels. */
+#include "logging_levels.h"
+
+/* Logging configuration for the HTTP library. */
+#define LIBRARY_LOG_NAME     "HTTP"
+#define LIBRARY_LOG_LEVEL    LOG_INFO
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/**
+ * @brief Server's root CA certificate for TLS server authentication.
+ *
+ * This certificate should be PEM-encoded.
+ */
+#define SERVER_CERTIFICATE                                             \
```

Comment:
IN PR #971 the MQTT server auth demo reads the cert from the file instead of pasting it in the code. Should the same thing be done here?

Reply:
Right, I initially had it like that too but had a discussion with Lee:
https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/971#discussion_r433391624

The trouble is that `fopen` uses current working directory as the base for the relative directory. We could `chdir` to the path of the executable, but the code we write to find that path might not always work depending on the OS.

Reply:
Nevermind. Lee and I agreed that it is troublesome for the user to have to add the key as a macro so we're back to opening the file using `fopen`.